a3p-integration/proposals/f:fast-usdc/deploy.test.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+/* global globalThis */
+import test from 'ava';
+import '@endo/init/legacy.js'; // axios compat
+import { makeVstorageKit } from '@agoric/client-utils';
+
+const io = { fetch: globalThis.fetch };
+const networkConfig = {
+  rpcAddrs: ['http://0.0.0.0:26657'],
+  chainName: 'agoriclocal',
+};
+
+test('fastUsdc is in agoricNames.instance', async t => {
```

Comment:
I guess this doesn't run until @agoric/synthetic-chain spins up the chain and stuff?

Reply:
That's right; `deploy.test.js` is invoked by `test.sh`, which is invoked in the `TEST` stage. See [docs on stages](https://github.com/Agoric/agoric-3-proposals/tree/main?tab=readme-ov-file#stages).